assets/src/scss/new-identity/style.scss

```diff
@@ -15,4 +15,16 @@
   --button-primary--color: var(--grey-900);
   --button-primary--visited--color: var(--grey-900);
   --button-primary--hover--color: var(--grey-900);
+  --button--focus--outline: 2px solid #99c3ff;
```

Comment:
This will impact all buttons, right? Not just post tags?

Reply:
Exactly! It's part of the [new layout for buttons](https://www.figma.com/file/9NtbY8n3at8uOEJTsLrETb/P4-Design-System?node-id=370-35690&t=whzv8fJGAqVBOf1L-4)

Reply:
Do we actually need to override it? It seems it already looks as expected in the current buttons and post tags

Reply:
Yeah I think we can directly change it in the typography file, since it should be applied regardless of the new identity! 🙂